api/authenticator.py

```diff
@@ -1541,9 +1541,10 @@ class BasicAuthenticationProvider(AuthenticationProvider, HasSelfTests):
         { "key": TEST_IDENTIFIER,
           "label": _("Test Identifier"),
           "description": _("A valid identifier that can be used to test that patron authentication is working."),
+          "required": True,
         },
         { "key": TEST_PASSWORD, "label": _("Test Password"), "description": _("The password for the test identifier."),
-          "optional": True,
+          "required": True,
```

Comment:
Did you mean to change this one? I think it needs to be optional because some libraries don't have passwords, only barcodes.

Reply:
Yes, I meant to remove and not change to `required`.